corehq/apps/reports/standard/monitoring.py

```diff
@@ -965,8 +965,12 @@ def get_row(self, user=None):
 
     def get_raw_user_link(self, user):
         from corehq.apps.reports.standard.inspect import SubmitHistory
+        sub_time_param = {
+            CompletionOrSubmissionTimeFilter.slug:
+            CompletionOrSubmissionTimeFilter.get_value(self.request, self.domain) or ''
+        }
         return _get_raw_user_link(user, SubmitHistory.get_url(domain=self.domain),
-                                  filter_class=EMWF)
+                                  filter_class=EMWF, additional_params=sub_time_param)
```

Comment:
Instead of additional parameters, should we be specific on what we are going to pass? I am not sure if we will ever need to pass in more paramters.

Reply:
I want to make `_get_raw_user_link` as general as possible, in case in the future we want to pass more parameters